contents/handbook/growth/sales/customer-onboarding.md

```diff
@@ -116,21 +119,20 @@ Every trial should have an end date by which time we expect the customer to make
 If they do become a customer (yay!) then we should agree a regular check in call cadence with them from the start (it's much harder to do after they are in the steady state).
 
 
-## Sales to CSM "handoff"
+## 1-hr onboarding call
 
-While we don't follow a traditional model of sales and customer success being fully separate roles with a formal "handoff", we are experimenting with spezialization within the CSM role. Simon is leading new acquisition, commercial conversations, evaluations (especially for larger enterprise-y customers), and Cameron is managing existing accounts, holding regular (at least quarterly) meetings with larger accounts, where review technical issues, product feedback/requests, and gauge overall account health in a more qualitative way to supplement product usage data in pocus and help retention.
+Customers on the Teams plan (and up) are entitled to a one hour kickoff/implementation call. This could include a high-level discussion of how PostHog fits into their stack, troubleshooting issues they've hit so far, or walking them through as they code it up, for simple setups. In practice we only need to worry about this for product-led / customers who haven't talked to sales before. [TO DO] Include a team calendly link in the welcome emails for Teams purchases or set up a separate campaign to email from a CSM. 
 
-In practice, everyone on the team should have context on high-touch customers from standups, but a formal introduction will be made when a Slack channel is set up. We don't currently have a great process for this for email only customers, but working this out is a to do (as is keeping better tabs on non-Slack high revenue customers more generally). Given the size of our team, we will still mostly need everyone to have context on all customers, so we have redundancy to cover absences.
+## Sales to CSM "handoff"
 
+Current thinking around this is based on customer spend. 
 
-## Meeting cadence and owners
+- < $20K AR: Sales process is self-serve, with some added assistance for Teams plan purchase (eg onboarding call)
+- $20K - $100K ARR:
+   - Cameron is primary for sales + onboarding to get them to 2 bills paid (or annual contract)
```

Comment:
Just so we're not maintaining everyone's roles in two places, I think you can delete this section and just [link to this?](https://posthog.com/handbook/growth/sales/overview#who-the-sales--cs-team-are)